package.json

```diff
@@ -18,6 +18,7 @@
     "dev:backend:realtime": "turbo dev --log-order stream --filter '@apps/hash-realtime' --",
     "dev:backend:search-loader": "turbo dev --log-order stream --filter '@apps/hash-search-loader' --",
     "dev:frontend": "turbo dev --log-order stream --filter '@apps/hash-frontend' --",
+    "dev:graph": "RUST_LOG=info yarn workspace @apps/hash-graph dev",
```

Comment:
Do we want to call it `start` instead? We don't have hot-reloading in Rust.

Reply:
I think it's probably easier to call it `dev` since we're also running `dev` for the frontend and Node API, and it saves having to remember 'oh yeah it's `start` for the graph'. I don't feel strongly about this though.

Reply:
I don't feel strongly either 🤷🏻‍♂️